### ISSUE TITLE
Email limitation

### ISSUE THREAD

**Issue by bloggerpact** [digitalpianism/abandonedcarts#22]
I don't know its's a bug or missing feature. plugin send email after 1 day ( Which I selected 1 day) to abandonment cart customer but the plugin sending same abandonment cart email to the same customer on daily basis again and again. does plugin has any option to set limit per customer? 

using Version number: 1.0.7

**Comment by digitalpianism** [comment]
@bloggerpact thanks for the report. Are you referring to the abandoned cart email or the sale abandoned cart email ?

**Comment by bloggerpact** [comment]
I'm referring to the abandoned cart email.  abandoned cart customer getting same abandonment cart email to on daily basis again and again. Please fix it

**Comment by digitalpianism** [comment]
@bloggerpact could you create the following test.php file in your root folder with the following code and give me the result when you access that file from your browser:

```
<?php
//some settings
error_reporting(E_ALL | E_STRICT);
define('MAGENTO_ROOT', getcwd());
$mageFilename = MAGENTO_ROOT . '/app/Mage.php';
require_once $mageFilename;
Mage::setIsDeveloperMode(true);
ini_set('display_errors', 1);
umask(0);
//instantiate the app model
Mage::app();

echo get_class(Mage::getResourceModel('sales/quote'));
```

**Comment by bloggerpact** [comment]
As you said to run the following code and i did this is what i got : 
DigitalPianism_Abandonedcarts_Model_Sales_Resource_Quote

**Comment by digitalpianism** [comment]
@bloggerpact thanks for that. That means it's not a conflict with another module. Let me investigate the issue and come back to you ASAP ;)

**Comment by bloggerpact** [comment]
I just have limited numbers of the plugin and yeah it's not a conflict with another module. plugin functionality working fine but when it's come to sending email have this issue. I hope you will fix it

**Comment by digitalpianism** [comment]
@bloggerpact will definitely fix it ;)

**Comment by digitalpianism** [comment]
@bloggerpact also can you confirm you don't have dryrun set to Yes ?

**Comment by digitalpianism** [comment]
@bloggerpact I'm unable to reproduce the issue on my side. Can you check your `sales_flat_quote` table and check if there are some rows with `abandoned_notified` equals 1 ?

**Comment by bloggerpact** [comment]
Yes! I'm using dry-run because some customer reported that your website sending me emails every day so I enabled dry-run.  I have checked sales_flat_quote table I didn't find any row with 1. every row has same value 0

**Comment by digitalpianism** [comment]
@bloggerpact basically dryrun set to Yes will not set the rows to 1. Once those rows are set to 1 the customer should not receive the email

**Comment by bloggerpact** [comment]
I didn't get you " those rows are set to 1 the customer should not receive the email" I just want to send once but not every day

**Comment by bloggerpact** [comment]
@digitalpianism I don't wanna stop email.

**Comment by digitalpianism** [comment]
@bloggerpact that's not what I meant. Basically when you have DryRun set to No, the module sets the abandoned_notified to 1 and thus this specific customer quote is not being notified again. With dryrun set to Yes, it should only work with test email.

Do you have the logs enabled ? Under Digital Pianism > Abandoned Carts > Logs. Also is there anything in `var/log/digitalpianism_abandonedcarts.log` ? Also check `var/log/system.log` and `var/log/exception.log` I feel like an exception is being thrown before the cart is marked as notified

**Comment by digitalpianism** [comment]
@septoctobre par hasard, vous n'auriez pas rencontré ce problème après les changements récents que nous avons effectué ?

**Comment by septoctobre** [comment]
@digitalpianism Non, cela fonctionne bien de mon côté. Il faudrait voir les logs !

**Comment by bloggerpact** [comment]
Hello,

there is something wrong with your plugin when I installed your plugin I was on dry run that was working fine but when I disabled dry run plugin doesn't send notifications :| I have tried to send manually using send button but result same.

![1](https://cloud.githubusercontent.com/assets/22036537/21578374/8acaecee-cfa3-11e6-96cb-f88c68f95d1b.PNG)
![2](https://cloud.githubusercontent.com/assets/22036537/21578364/34993b0a-cfa3-11e6-80dc-6b79aff9103b.PNG)
![3](https://cloud.githubusercontent.com/assets/22036537/21578366/3890610c-cfa3-11e6-80ad-c93dd8896064.PNG)

**Comment by digitalpianism** [comment]
@bloggerpact are those customers in the Retailer customer group ?
